### PR TITLE
fix kubeadm typos

### PR DESCRIPTION
**Pull request by jdcloudconformance**
#### What type of PR is this?
/kind cleanup

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: cmd/kubeadm/app/apis/bootstraptoken/v1/defaults.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ import (
 
 const (
 	// DefaultTokenDuration specifies the default amount of time that a bootstrap token will be valid
-	// Default behaviour is 24 hours
+	// Default behavior is 24 hours
```
Comment (xuzhenglun) [inline cmd/kubeadm/app/apis/bootstraptoken/v1/defaults.go:28]: Behaviour and behavior are two spellings of the same noun. The spelling depends on the type of English. In British English, you use “behaviour.” In American English, you use “behavior.”

I don't think this need to be correct.

**Comment by xuzhenglun** [comment]
Thanks for you contribution

/ok-to-test
/triage accepted
/priority backlog
/release-note-none

File: cmd/kubeadm/app/apis/bootstraptoken/v1/defaults.go (hunk 1 above)
Comment (neolit123) [reply cmd/kubeadm/app/apis/bootstraptoken/v1/defaults.go:28]: we tend to use USA english

**Review by neolit123 — COMMENTED** [review]
are these all the typos in kubeadm, currently?

**Review by neolit123 — COMMENTED** [review]
someone sent a bigger PR
https://github.com/kubernetes/kubernetes/pull/127079

/close
thanks

**Comment by neolit123** [comment]
> are these all the typos in kubeadm, currently?

i asked the Q here but got no response ^

**Pull request closed without merging**